src/vision/status_quo/barbara_carefully_dismisses_embedded_future.md

```diff
@@ -246,6 +246,49 @@ instead. Here are some ways in which these APIs are lighter weight than a
       community and ecosystem.
     * [Niklaus] would really have struggled. If he asked for help, he probably
       would've gotten conflicting advice from the community.
+* **`Future` has a lot of features that Barbara's traits don't have -- aren't
+    those worth the cost?**
+    * `Future` has a lot of additional features that are nice-to-have. However,
```

Comment:
I'd sort of like to know what these features are, can they be concisely listed?

Reply:
I added a list. The list has a lot in common with Aaron Turon's [blog post](http://aturon.github.io/tech/2016/09/07/futures-design/) describing the initial design of `Future`.